panic on `status tx` with bad tx ID

```
➜  smrepl git:(master) ./smrepl_darwin_amd64
<APP> INFO 001 02:40:07.463 repl.Start ▶ new session started
Welcome to Spacemesh. Connected to api server at localhost:9092 (GRPC API 1.1). >> Insecure Connection. Use only with a local trusted server <<
Network info:
Network id: 204
Layers per epoch: 6
Layer duration: 200 seconds
Current layer: 988
Current epoch: 164
Genesis time: 2021-09-13 19:49:40 +0300 MSK
$ status tx
$ Enter transaction id: 0x76cf0c12f500fa114cea590ac686bb147fe589d9ff4a44b3646d2bd90df839ef
State: Processed
Transaction summary:
Id: 0x76cf0c12f500fa114cea590ac686bb147fe589d9ff4a44b3646d2bd90df839ef
From: 0xfF083C9A22E05D3459B03f3DbeD61c7AD6E0a209
To (coin account): 0x0000000000000000000000000000000000000001
Nonce: 0
Amount: 10 Smidge
Fee: 100 Smidge
$ status tx
$ Enter transaction id: 123
State: Unspecified state
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x15b0dbe]

goroutine 1 [running]:
github.com/spacemeshos/smrepl/repl.printTransaction(0xc0000e42a0)
	/Users/nkryuchkov/GolandProjects/smrepl/repl/transactions_commands.go:178 +0x3e
github.com/spacemeshos/smrepl/repl.(*repl).printTransactionStatus(0xc000524180)
	/Users/nkryuchkov/GolandProjects/smrepl/repl/transactions_commands.go:44 +0x19c
github.com/spacemeshos/smrepl/repl.(*repl).executor(0xc000524180, 0xc000038520, 0x9)
	/Users/nkryuchkov/GolandProjects/smrepl/repl/repl.go:239 +0x227
github.com/c-bata/go-prompt.(*Prompt).Run(0xc000528300)
	/Users/nkryuchkov/go/pkg/mod/github.com/c-bata/go-prompt@v0.2.3/prompt.go:85 +0x7b0
github.com/spacemeshos/smrepl/repl.runPrompt(0xc0005084a0, 0xc0005084b0, 0xc00059be40, 0x21)
	/Users/nkryuchkov/GolandProjects/smrepl/repl/prompt.go:25 +0x110
github.com/spacemeshos/smrepl/repl.Start(0x17f8728, 0xc00043ab40)
	/Users/nkryuchkov/GolandProjects/smrepl/repl/repl.go:221 +0x298
main.main()
	/Users/nkryuchkov/GolandProjects/smrepl/main.go:52 +0x3f1
➜  smrepl git:(master) 
```

--- a/repl/transactions_commands.go
+++ b/repl/transactions_commands.go
@@ -3,14 +3,14 @@ package repl
 import (
 	"encoding/hex"
 	"fmt"
-	"github.com/spacemeshos/ed25519"
 	"strconv"
 
 	apitypes "github.com/spacemeshos/api/release/go/spacemesh/v1"
-	"github.com/spacemeshos/go-spacemesh/common/util"
+	"github.com/spacemeshos/ed25519"
+	"github.com/spacemeshos/smrepl/log"
 
 	gosmtypes "github.com/spacemeshos/go-spacemesh/common/types"
-	"github.com/spacemeshos/smrepl/log"
+	"github.com/spacemeshos/go-spacemesh/common/util"
 )
 
 var transactionStateDisStringsMap = map[int32]string{
@@ -40,7 +40,7 @@ func (r *repl) printTransactionStatus() {
 		fmt.Println("Unknown transaction state")
 	}
 
-	if tx != nil {
+	if tx != nil && tx.Id != nil {
 		printTransaction(tx)
 	} else {
 		fmt.Println("Unknown transaction")

